features/share/graphql/graphql-enhancements/node/package-lock.json

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
I wouldn't include a package-lock.json for the demo, you already have requirements soft-pinned in package.json